src/Helper/EnumTypeHelper.php

```diff
@@ -2,19 +2,19 @@
 
 namespace ShipMonk\PHPStan\Helper;
 
-use PHPStan\Type\ArrayType;
 use PHPStan\Type\IntersectionType;
 use PHPStan\Type\Type;
-use PHPStan\Type\TypeWithClassName;
 use PHPStan\Type\UnionType;
 
 class EnumTypeHelper
 {
 
     public static function containsEnum(Type $type): bool
     {
-        if ($type instanceof ArrayType && self::containsEnum($type->getItemType())) {
-            return true;
+        foreach ($type->getArrays() as $arrayType) {
+            if (self::containsEnum($arrayType->getItemType())) {
```

Comment:
You could do `$type->isArray()->yes() && $type->getIterableValueType()->isEnum()->yes()`.